frontend/src/lib/components/report/elements/SliceElement.svelte

```diff
@@ -17,20 +18,23 @@
 	export let element: ReportElement;
 
 	const zenoClient = getContext('zenoClient') as ZenoService;
+
 	let sliceElementSpec: SliceElementSpec | undefined;
 	let sliceElementOptions: SliceElementOptions | undefined;
 	let table: Record<string, string | number | boolean>[] | undefined = [];
 	let page = 0;
 
 	$: updateSliceElementSpec(element.data as string);
+	$: viewSchema = JSON.parse(sliceElementOptions?.project.view || '{}') as ViewSchema;
```

Comment:
This will fail if the JSON.parse fails because it can't be cast. Will probably crash frontend then.

Reply:
Won't crash b/c it's typescript, but maybe cleaner way to do this?